crucible-mir-comp/src/Mir/Compositional/Builder.hs

```diff
@@ -742,6 +751,8 @@ regToSetup bak p eval shp rv = go shp rv
         refSV <- go refShp refRV
         lenSV <- go lenShp lenRV
         pure $ MS.SetupSlice $ MirSetupSliceRaw refSV lenSV
+    go (EnumShape _ _ _ _ _) _ =
+      error "Enums not currently supported in overrides"
```

Comment:
See https://github.com/GaloisInc/saw-script/issues/1990 for the task of supporting enums in `{crucible,crux}-mir-comp`.